### DISCUSSION TITLE
Grafana - MySQL compatibility matrix

### DISCUSSION THREAD

**Discussion by nvsajeeva** [grafana/grafana#59192] [General]
Hello, 
I would like to know if Grafana v6.1.6 is compatible with MySQL 8 as the backend database. I haven't been able to find any documentation related to this specific requirement. 
Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
